src/ui-kit/Pagination/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setCurrentPage } from 'redux/actions'
+import { Pagination } from 'antd'
+import './pagination.scss'
+
+const PaginationKit = ({setCurrentPage, pageCurrent}) => {
+
+  const setCurrentPageHandler = (page) => {
```

Comment:
did you run prettier ? I think if you have 1 parameter it will remove the () braces, I just dont think that you did run prettier....